src/transaction/flashback.c

```diff
@@ -55,6 +55,177 @@
 #include "thread_entry.hpp"
 #include "oid.h"
 #include "storage_common.h"
+#include "system_parameter.h"
+
+static volatile LOG_PAGEID flashback_Min_log_pageid = NULL_LOG_PAGEID;	// Minumun log pageid to keep archive log volume from being removed
+static volatile time_t flashback_Last_request_done_time = -1;	// The time most recently requested by flashback
+static volatile int flashback_Threshold_to_remove_archive = 0;	/* If the difference between the time at which the archive log is deleted
+								 * and the time the flashback last requested exceeds this threshold,
+								 * the archive log volume can be deleted.
+								 */
+// *INDENT-OFF*
+static std::atomic_bool flashback_Is_active = false;	// the status value that the flashback is processing the request
+// *INDENT-ON*
+
+/*
+ * flashback_set_min_log_pageid_to_keep - set flashback_Min_log_pageid
+ */
+
+void
+flashback_set_min_log_pageid_to_keep (LOG_LSA * lsa)
+{
+  assert (lsa != NULL);
+
+  flashback_Min_log_pageid = lsa->pageid;
+}
+
+/*
+ * flashback_unset_min_log_pageid_to_keep - unset flashback_Min_log_pageid to NULL_LOG_PAGEID
+ */
+
+void
+flashback_unset_min_log_pageid_to_keep ()
+{
+  flashback_Min_log_pageid = NULL_LOG_PAGEID;
+}
+
+/*
+ * flashback_min_log_pageid_to_keep - returns minimum log pageid to keep
+ *
+ * return   : minimum pageid that flashback have to keep
+ */
+
+LOG_PAGEID
+flashback_min_log_pageid_to_keep ()
+{
+  return flashback_Min_log_pageid;
+}
+
+/*
+ * flashback_set_request_done_time - set flashback_Last_request_done_time to current
+ */
+
+void
+flashback_set_request_done_time ()
+{
+  flashback_Last_request_done_time = time (NULL);
+}
+
+/*
+ * flashback_set_threshold - set flashback_Threshold_to_remove_archive
+ *                           it follows PRM_ID_REMOVE_LOG_ARCHIVES_INTERVAL
+ */
+
+static void
+flashback_set_threshold ()
+{
+  flashback_Threshold_to_remove_archive = prm_get_integer_value (PRM_ID_REMOVE_LOG_ARCHIVES_INTERVAL);
+  if (flashback_Threshold_to_remove_archive < 60)
```

Comment:
1. Is this value, 60, fixed(maybe seconds? or ms?)? If not, what about expression macro? I suggest `THRESHOLD_LIMIT_SECOND`.
3. Where can I find the reason of this policy(60)?

Reply:
Thank you for feedback, 
1. I will define constant value for it
2. 60 is the recommended value for system parameter "remove_log_archive_interval_in_secs" 
    in manual. (https://www.cubrid.org/manual/en/11.0/admin/config.html#logging-related-parameters)

Reply:
Oh, I see. 
Thank you for the link. I checked it.